MetaMask/metamask-extension
# Updating ppom validator

## **Description**

Updating PPOM Validator package.
We have done performance improvements in PPOMController for mobile device, we can leverage same for extension also.

## **Related issues**

Fixes: NA

## **Manual testing steps**

1. Go to test DAPP
2. Submit a malicious transaction
3. It should show error on page

## **Screenshots/Recordings**

### **Before**

N/A

### **After**

N/A

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
